packages/builder/src/components/userInterface/ComponentsHierarchy.svelte

```diff
@@ -57,7 +61,8 @@
     {#if $store.currentPreviewItem.props._instanceName && $store.currentPreviewItem.props._instanceName === screen.props._instanceName && screen.props._children}
       <ComponentsHierarchyChildren
         components={screen.props._children}
-        currentComponent={$store.currentComponentInfo} />
+        currentComponent={$store.currentComponentInfo}
+        dragDropStore={writable({})} />
```

Comment:
Will this store not be reinstantiated every time the component renders? Could we just use local state inside the component?

Reply:
we could do that, cant see it making any difference, but i will change

Reply:
That’s not what I meant. Local state is like `let dragDropState = {}`. I was wondering why you used a store here inside a single component. If you are going to be cheeky, at least be correct :)